src/main/java/io/phasetwo/service/resource/RolesResource.java

```diff
@@ -41,12 +44,52 @@ public Stream<OrganizationRole> getRoles() {
   @POST
   @Consumes(MediaType.APPLICATION_JSON)
   public Response createRole(OrganizationRole representation) {
-    if (!auth.hasManageOrgs() && !auth.hasOrgManageRoles(organization)) {
-      throw new NotAuthorizedException(
-          String.format(
-              "User %s doesn't have permission to manage roles in org %s",
-              auth.getUser().getId(), organization.getName()));
-    }
+    canManage();
+
+    OrganizationRole or = createOrganizationRole(representation);
+
+    return Response.created(
+            session.getContext().getUri().getAbsolutePathBuilder().path(or.getName()).build())
+        .build();
+  }
+
+  @PUT
+  @Consumes(MediaType.APPLICATION_JSON)
+  public Response createRoles(List<OrganizationRole> representation) {
+    canManage();
+
+    representation.forEach(this::createOrganizationRole);
```

Comment:
Need a test that validates when some of the creates fail:
1. everything fails
2. no events get emitted
3. a proper error gets sent back